reV/generation/generation.py

```diff
@@ -97,9 +103,105 @@ class Gen:
                  'lcoe_nom': {'scale_factor': 1, 'units': 'dol/MWh',
                               'dtype': 'float32', 'chunks': None,
                               'type': 'scalar'},
-                 'lcoe_fcr': {'scale_factor': 1, 'units': 'dol/MWh',
+                 'lcoe_fcr': {'scale_factor': 1, 'units': 'dol/mwh',
                               'dtype': 'float32', 'chunks': None,
                               'type': 'scalar'},
+                 # Solar water heater
+                 'T_amb': {'scale_factor': 1, 'units': 'C',
```

Comment:
This is getting to be a lot of outputs! Which is good. But it's getting cumbersome. @MRossol, do you think we should put this in a json? If we put it in the same dir as this file, we can access it using os.path.dirname(__file__), right?

Reply:
I like that idea. We could add an outputs.json to reV/generation and reV/econ and handles the output dictionaries and load them into the module.

If we do that I'd also suggest that we set the path as a cls variable or kwarg which would allow users to provide a custom json (unless that will break $hit)

Reply:
@grantbuster  and @mikebannis  do we want to implement this here or in a new PR?

Reply:
I would say new PR. The current changes need to get pulled into reV for Parthiv